MHacks/AnnouncementCell.swift

```diff
@@ -16,6 +16,8 @@ class AnnouncementCell: UITableViewCell {
     @IBOutlet weak var dateLabel: UILabel!
     @IBOutlet weak var messageLabel: UILabel!
 	@IBOutlet weak var colorView: UIView!
+	@IBOutlet weak var sponsoredLabel: UIView!
+	@IBOutlet weak var contentStackView: UIStackView!
```

Comment:
Please do not use weak for any of these vars and remove it from the ones that already exist. There's no need for these views to be weakly held, and it's a bug if they actually get released. The Sponsored label getting released across cell reuse caused a crash for me.